bigtable/gc.go

```diff
@@ -25,10 +25,21 @@ import (
 	bttdpb "google.golang.org/genproto/googleapis/bigtable/admin/v2"
 )
 
+const (
+	PolicyType_None = iota
+	PolicyType_MaxAge
+	PolicyType_MaxVersion
+	PolicyType_Union
+	PolicyType_Intersection
+)
+
+type PolicyType = int
+
 // A GCPolicy represents a rule that determines which cells are eligible for garbage collection.
 type GCPolicy interface {
 	String() string
 	proto() *bttdpb.GcRule
+	PolicyType() PolicyType
```

Comment:
Apology for my late update, it's been busy in the last few days. I have added a wrapper struct called TypedGCPolicy so that it won't break existing user of this client.